cmd/frontend/main.go

```diff
@@ -9,5 +10,6 @@ import (
 // function for details.
 
 func main() {
+	go authz.SetProviders(true, []authz.Provider{})
```

Comment:
I think it is safe without making a goroutine (as OSS does not have authz providers nor does we watch for config changes to them):
```suggestion
	// Set dummy authz provider to unblock channel for checking permissions in GraphQL APIs.
	// See https://github.com/sourcegraph/sourcegraph/issues/3847 for details.
	authz.SetProviders(true, []authz.Provider{})
```

Reply:
cc @beyang to suggest wording here.